### ISSUE TITLE
Replace underscores with hyphens in set names

### ISSUE THREAD

**Issue by NateLanza** [visdesignlab/upset-alt-txt-gen#46]
Underscores are read out by screen readers, making them cumbersome to have within alttxts. However, spaces split the set name into two words, which makes navigating the text by word slightly slower. Hyphens are not read out by screen readers but do maintain a set name as a single word, so underscores in set names should be replaced by hyphens.
